hack/generated/controllers/controller_test.go

```diff
@@ -123,10 +122,7 @@ func StorageAccount_BlobServices_CRUD(t *testing.T, testContext testcommon.KubeP
 	g := NewGomegaWithT(t)
 
 	blobService := &storage.StorageAccountsBlobService{
-		ObjectMeta: ctrl.ObjectMeta{
-			Name:      "default", // MUST be 'default'
```

Comment:
this is the thing I’ve been working towards 😀

Reply:
(I know @babbageclunk likes stuff like this 😉)

Reply:
It's like when they say the title of the movie in the movie.